airflow/config_templates/default_airflow.cfg

```diff
@@ -422,6 +422,13 @@ backend =
 # ``{{"connections_prefix": "/airflow/connections", "profile_name": "default"}}``
 backend_kwargs =
 
+# Advanced secrets backend configuration, allow to user configure more than one secret backend,
+# order of secrets backend and turn off built-in backends.
+# Expected JSON list of objects. See
+# https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/index.html#advanced-configuration
+# for more details.
+backends_config =
```

Comment:
I think it woudl be nice to add the default way of configuration as config. It would give the user an opportunity to figure out how to do it without reaching to documentation. 

There are two kinds of users, those who love reading the docs, and those who prefer to explore. Comment (or even default value written here would actually be nice one for the latter group.

Reply:
Oh... default value for current implementation - "not defined", null or None and fallback to current implementation.

However if we decide to keep only one way to configure backends in this case we might set as default:
```ini
backends_config = [{"backend":"airflow.secrets.environment_variables.EnvironmentVariablesBackend"}, {"backend":"airflow.secrets.metastore.MetastoreBackend"}]
```

Reply:
> This might happen that someone will set the three of them, and backends_config should get precedence

I can't find the good way how to do it right now with `AirflowConfigParser.deprecated_options`, because if we put this configs as deprecated and try to check is user actually put something in this config the result will - deprecation warning every time even if user do not use that.

I think I do not try only one option - get this configs without fallback, in this case probably we only have to catch exception and no deprecation warnings raised in case if user do not set anything. It just my assuming.

Another interesting moment it would a first time when one config option replace two options but right now all deprecation expected one-to-one